api/config/config_local.json

```diff
@@ -18,6 +18,6 @@
   },
   "server": {
     "stackdriverApiKey": "",
-    "projectId": ""
+    "projectId": "all-of-us-workbench-test"
```

Comment:
So the other place project ID gets used appears to be Stackdriver logging... I'm not sure what this does when you're in dev... specifying all-of-us-workbench-test there may not be correct.

Furthermore, the project that CDR versions are in for BigQuery is in isn't necessarily the project for the app generally... I think we're conflating two different things here?

You may just want to pull the project for the CDR from the default CDR version instead.